app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1006,6 +1007,12 @@ class MultipleDisposalsTriageController @Inject() (
             fb => fb._2.fold(_.representeeAnswers, _.representeeAnswers)
           )
 
+        val isFurtherReturn = state
+          .fold(
+            _.representeeAnswers.map(_.fold(_.isFirstReturn.contains(false), _.isFirstReturn)).contains(false),
```

Comment:
this part of the fold for `StartingNewDraftReturn` is only looking at  representee answers - it won't work for people who aren't representatives. If you look in `isFurtherReturn` in `FillingOutReturn` you'll see the kind of logic you'll need for `StartingNewDraftReturn` - you could reuse a lot of it.

Reply:
looks like you've already written `isFurtherReturn` for `StartingNewDraftReturn` but you just haven't used it